xmake/modules/package/manager/cmake/find_package.lua

```diff
@@ -100,6 +101,10 @@ function _find_package(cmake, name, opt)
 
     -- run cmake
     local envs = configs.envs or opt.envs
+    -- decide whether to enable debug build type
+    if (not envs.CMAKE_BUILD_TYPE and configs.debug) then
```

Comment:
use `opt.mode == "debug"`

Reply:
This is done. But `opt.mode` seems always to be `release`